ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -480,6 +481,36 @@ Rectangle {
         return found[1]
     }
 
+    function getTextWithPublicKeys() {
+        let result = messageInputField.getText(0, messageInputField.length)
+
+        if (mentionsPos.length > 0) {
+            let lastIndex = messageInputField.length + 1
+            while (lastIndex > 0) {
+
+                let currentMax = -1
+                let maxIndex
+
+                for (var i = 0; i < mentionsPos.length; i++) {
+                    if (mentionsPos[i].rightIndex < lastIndex && mentionsPos[i].rightIndex > currentMax) {
+                        currentMax = mentionsPos[i].rightIndex
+                        maxIndex = i
+                    }
+                }
+
+                lastIndex = 0
+                if (currentMax >= 0) {
+                    result = result.substring(0, mentionsPos[maxIndex].leftIndex + 1)
```

Comment:
@faust4exe theoretically `maxIndex` is not initialized in line `492` and accessing it here may be `undefined` value.

Reply:
That is true - but currentMax will be -1 in that case. But ok - will initialize it